### PR TITLE
add license header to source files

### PR DESCRIPTION
**Pull request by hellochar**
now that we release modules, all source files (that get compiled into
the /build folder) should have license headers.

### CODE REVIEW

File: license_header.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*!
 Plottable @VERSION (https://github.com/palantir/plottable)
-Copyright 2014-2017 Palantir Technologies
+Copyright 2014-present Palantir Technologies
```
Comment (CalvinFernandez) [inline license_header.txt:3]: is this correct?

Comment (hellochar) [reply license_header.txt:3]: Yeah I think "present" is an acceptable thing to put here; @themadcreator @adidahiya ?

Comment (themadcreator) [reply license_header.txt:3]: Facebook uses this style in their react headers: e.g. https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react.js

**Review by themadcreator — APPROVED** [review]
I kind of wanted:
```
/**
 * Copyright ...
 */
```
but this is fine

**Review by adidahiya — CHANGES_REQUESTED** [review]
if we're making changes here, might as well do JSDoc license headers. http://usejsdoc.org/tags-license.html

**Comment by hellochar** [comment]
@adidahiya @themadcreator how's that?

**Review by adidahiya — APPROVED** [review]

**Pull request merged**